server/statistics/metrics.go

```diff
@@ -127,6 +127,29 @@ var (
 			Help:      "Bucketed histogram of the batch size of handled requests.",
 			Buckets:   prometheus.LinearBuckets(0, 5, 12),
 		})
+
+	regionAbnormalPeerDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "pd",
+			Subsystem: "regions",
+			Name:      "abnormal_peer_duration_seconds",
+			Help:      "Bucketed histogram of processing time (s) of handled success cmds.",
+			Buckets:   prometheus.ExponentialBuckets(1, 1.4, 30), // 1s ~ 6.72 hours
+		}, []string{"type"})
+
+	regionIsDangerousCounter = prometheus.NewCounter(
```

Comment:
I think pd cannot know the abnomal region because the region cannot elect the leader and report to PD.

Reply:
Great catch.